package.json

```diff
@@ -29,8 +29,5 @@
     "eslint": "^6.8.0",
     "expo-yarn-workspaces": "*",
     "prettier": "^1.19.0"
-  },
-  "resolutions": {
-    "@react-native-community/cli-platform-ios": "3.0.0"
```

Comment:
It's probably safe. As far as I know `@r-n-c/cli-platform-ios` there not this much that this change could break.